### PR TITLE
Remove banner

### PR DESCRIPTION
**Pull request by astrofrog**
While this banner was added at the peak of the events earlier this year, it might be time to remove this banner and perhaps replace it with a more general statement such as the one suggested here

Screenshot 2020-10-09 at 10 59 09

### CODE REVIEW

**Review by taldcroft — APPROVED** [review]
LGTM

File: roles.json [hunk 1]
```diff
@@ -267,6 +267,25 @@
             ]
         }
     },
+    {
+        "role": "DevOps and Operations",
```
Comment (pllim) [inline roles.json:271]: This seems unrelated. I thought this is being discussed over at #400

Comment (astrofrog) [reply roles.json:271]: Oops fail

**Review by pllim — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -91,16 +91,14 @@
...
-    </section>
-
 	<section id="hero">
 		<img src="images/astropy_project_logo.svg" width="524" onerror="this.src='images/astropy_project_logo.png'; this.onerror=null;" alt="Astropy: a community Python library for Astronomy" />
 		<p>The Astropy Project is a community effort to develop a <a href="http://docs.astropy.org">common core
 		package</a> for Astronomy in Python and foster an ecosystem of <a href="affiliated/index.html">interoperable
 		astronomy packages</a>.</p>
-		<p class="acknowledge">Please remember to <a href="acknowledging.html">acknowledge and cite</a> the use of Astropy!</p>
+        <p>The Astropy community <a href="https://www.astropy.org/code_of_conduct.html">is committed to
+        supporting diversity and inclusion</a></p>
```
Comment (eteq) [inline index.html:100]:
```suggestion
        <p>The Astropy community <a href="https://www.astropy.org/code_of_conduct.html">is committed to
        supporting diversity and inclusion</a>.</p>
```

**Review by eteq — APPROVED** [review]
I'm ok with this too since the original banner was in response to current events.  It's possible we want to follow-up with additional timeless content more specific and focused on our approach to combating racism (open to ideas about where that best fits!), but this is a good starting point.
